src/core/DY.Core.Bytes.fst

```diff
@@ -329,6 +346,26 @@ let get_dh_usage #cusages pk =
   | DhPub sk -> get_usage sk
   | _ -> NoUsage
 
+/// Obtain the label of the corresponding KEM private key of a KEM public key.
+/// Although the KEM public key label is public,
+/// this is useful to reason on the corresponding KEM private key label.
+
+[@@"opaque_to_smt"]
+val get_kem_label: {|crypto_usages|} -> bytes -> GTot label
+let get_kem_label #cusages pk =
```

Comment:
I think we should name this function `get_kem_sk_label` to better describe what the function does.

Reply:
Done, and we will need to give the same treatment to similar DH functions at some point